src/components/__tests__/Header.test.js

```diff
@@ -0,0 +1,30 @@
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import Header from "../Header/Header";
+import React from "react";
+
+test("renders Header component without crashing", () => {
+  render(<Header />);
+});
+
+test("renders all navigation links", () => {
+  const { getByText } = render(<Header />);
```

Comment:
Get elements by test id, avoid using get by text (and similar) as much as possible.